core/src/main/java/org/apache/calcite/rel/core/TableFunctionScan.java

```diff
@@ -195,6 +195,9 @@ public RelWriter explainTerms(RelWriter pw) {
     if (elementType != null) {
       pw.item("elementType", elementType);
     }
+    if (columnMappings != null) {
+      pw.item("columnMappings", columnMappings);
+    }
     return pw;
   }
```

Comment:
Would it bring some advantages?

Reply:
> Would it bring some advantages?
should not add in explainTerms.
How about add columnMappings in digest method.